twitter-export-image-fill.py

```diff
@@ -122,6 +141,11 @@
       if (not args.include_retweets) and retweeted:
         continue
 
+      # Download avatar for tweet and retweet
```

Comment:
Ok, that simplifies things a bit. I'll remove the count and that part of the message.